routes/tasks.ts

```diff
@@ -38,7 +38,7 @@ router.post("/", async (req: express.Request, res: express.Response) => {
     try {
         const createNewTask: Task = await req.body
         const taskId = await database.createTask(createNewTask)
-        res.status(200).send()
+        res.status(200).json(taskId)
```

Comment:
what is being sent back here?

Reply:
ID that's being generated on the backend